app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -54,6 +49,26 @@ public ApacheHttpClientMediaWikiApi(String apiURL) {
         api = new CustomMwApi(apiURL, httpClient);
     }
 
+    @Override
+    public String getEditToken() throws IOException {
```

Comment:
JavaDocs

Reply:
Ashish I have not added this function. I have added a relevant java docs to it. However I see that this function isn't being used anywhere, Is there any use of this function in future or shall I remove it?

Reply:
Ashish I have not added this function. I have added a relevant java docs to it. However I see that this function isn't being used anywhere, Is there any use of this function in future or shall I remove the function?